content/_code-samples/escrow/py/cancel_escrow.py

```diff
@@ -0,0 +1,32 @@
+from xrpl.models import EscrowCancel
+from xrpl.clients import JsonRpcClient
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to the testnetwork
+
+# Cancel an escrow
+# If the escrow wasnt created with a CancelAfter time, it never expires and this transaction fails
+
+escrow_creator = generate_faucet_wallet(client=client).classic_address # escrow creator/ sender address
+escrow_seq = int # escrow sequence
```

Comment:
Where does `int` come from?

Reply:
(The code should run on it's own without any changes, and also highlight how someone could modify one line or two to use it for their own similar use case)

Reply:
In this case, you'll probably want to send an EscrowCreate, then follow it up with EscrowCancel

Reply:
there is already an escrow create file is there any need to add another one to this file